Description: Fix manpage: escape minus signs, otherwise interpreted as hyphens by groff
Author: Jonas Meurer <jonas@freesources.org>
Last-Update: 2011-05-11

--- a/man/cryptsetup.8
+++ b/man/cryptsetup.8
@@ -171,7 +171,7 @@
 
 N.B. If key file is in GPG encrypted format, you have to use
 \-\-key-file=- and decrypt it before use.
-gpg --decrypt <keyfile> | cryptsetup loopaesOpen \-\-key-file=- <device> <name>
+gpg \-\-decrypt <keyfile> | cryptsetup loopaesOpen \-\-key-file=- <device> <name>
 
 Use \fB\-\-key-file\fR to specify proper key length, default compiled-in
 parameters are visible in \fB\-\-help\fR output.
@@ -441,7 +441,7 @@
 keyslot.
 
 Second type is used for volume (master) key. You can switch between
-using /dev/random and /dev/urandom  here, see \fP--use-random\fR and \fP--use-urandom\fR
+using /dev/random and /dev/urandom  here, see \fP\-\-use-random\fR and \fP\-\-use-urandom\fR
 options. Using /dev/random on system without enough entropy sources
 can cause \fPluksFormat\fR to block until the requested amount of random data is gathered.
 See \fPurandom(4)\fR for more information.
